### ISSUE TITLE
data_processngsim

### ISSUE THREAD

**Issue by Alreall** [dvij542/trajectory-prediction-GRIP#5]
Hi, thank you for the program, I noticed that you made changes to the original program to make it work for the ngsim dataset, but I have a few questions when I look at your code: (1) the sampling frequency of the ngsim dataset is 10hz, here you use 2 frames as one second, did you do a resampling of the original dataset, and how did you divide the training and test sets? (2) In your program, you provide a file after preprocessing, but the preprocessing file you provide is only a .pkl file, and there is no preprocessed training set data and test set data mentioned in the program.

**Comment by ontoon** [comment]
@Alreall 
Hi, May I ask if you have solved the problem of dividing training set and test set? If possible, can you help me solve it?
